test/unit_test/passes/onnx/test_add_metadata.py

```diff
@@ -0,0 +1,255 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from pathlib import Path
+from unittest.mock import patch
+
+import onnx
+import pytest
+
+from olive.passes.olive_pass import create_pass_from_dict
+from olive.passes.onnx.add_metadata import AddOliveMetadata
+from test.unit_test.utils import create_onnx_model_file, delete_onnx_model_files, get_onnx_model
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_onnx_model():
+    """Create ONNX model file for testing."""
+    create_onnx_model_file()
+    yield
+    delete_onnx_model_files()
```

Comment:
This module level `delete_onnx_model_files()` will delete the onnx model file for other tests. We only create onnx model for test once here: https://github.com/microsoft/Olive/blob/712d55bfc560581471fefcad6417727412ac31ee/test/unit_test/conftest.py#L13. Deleting it in this test will fail all other tests. You can just remove this `setup_onnx_model()`.